### PR TITLE
feat(record): add API to get period time-series data

### PR DESCRIPTION
**Pull request by atsushi421**
Signed-off-by: atsushi421 <a.yano.578@ms.saitama-u.ac.jp>

This PR add API to get period time-series data in record package.
The implementation is a modification of the following parts of the plot package.
https://github.com/tier4/CARET_analyze/blob/3a1ed200e8206fe4ef7eea184325e314b597c493/src/caret_analyze/plot/bokeh/callback_info.py#L122

To reproduce: https://drive.google.com/drive/folders/1BV7G34uA7cLB6XoN9xj7NAjOboh-rxmC?usp=sharing

### CODE REVIEW

File: src/caret_analyze/record/period.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            - {period_column}
+
+        """
+        records = self._create_empty_records()
+        if not self._target_timestamps:
+            return records
+
+        for i in range(1, len(self._target_timestamps)):
+            records.append(RecordFactory.create_instance(
+                {self._target_column: self._target_timestamps[i],
```
Comment (hsgwa) [inline src/caret_analyze/record/period.py:67]: I think left column should have former timestamp to make it the same as other APIs (e.g. response time, latency)

```suggestion
                {self._target_column: self._target_timestamps[i-1],
```

Comment (atsushi421) [reply src/caret_analyze/record/period.py]: @hsgwa 
Thank you very much for pointing that out.
I too thought former timestamps should be used, so I modified it: https://github.com/tier4/CARET_analyze/pull/168/commits/2f43a1ddd53a9aa13e188902103db5207e947005

[The previous implementation](https://github.com/tier4/CARET_analyze/blob/3a1ed200e8206fe4ef7eea184325e314b597c493/src/caret_analyze/plot/bokeh/callback_info.py#L132) used a later timestamp, and this modification changes the output results.
So I will notify the user when this Period API is used in the output of the Plot package.

**Comment by hsgwa** [comment]
@atsushi421 
Because https://github.com/tier4/CARET_analyze/pull/167 is merged, `__init__.py` occurs merge conflict.
Could you fix this, please?

**Comment by atsushi421** [comment]
@hsgwa 
I have resolved conflicts. :bow: 
https://github.com/tier4/CARET_analyze/pull/168/commits/daf4afde923affc79e21efef2d7bd0a7886b2739

**Review by hsgwa — APPROVED** [review]
LGTM!

**Pull request merged**